include/swift/SIL/SILPrintContext.h

```diff
@@ -29,6 +29,27 @@ class SILBasicBlock;
 /// Used as context for the SIL print functions.
 class SILPrintContext {
 public:
+  enum class Flag {
+    None,
+
+    /// Dump more information in the SIL output.
+    Verbose = 0x1,
+
+    /// Sort all kind of tables to ease diffing.
+    SortedSIL = 0x2,
+
+    /// Print debug locations and scopes.
+    DebugInfo = 0x4,
+
+    /// See \ref FrontendOptions.PrintFullConvention.
+    PrintFullConvention = 0x8,
+
+    /// Print out all types that normally would be surpressed.
+    PrintAllTypes = 0x16,
```

Comment:
`0x10`?

Reply:
You are right. Thanks! I was thinking 16 without hex.